### PR TITLE
[IMP] inventory: udpate shipments & deliveries doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] to the `shipments_deliveries` doc in Odoo's Inventory documentation, specific to 17.0.

Here's what I did:
- Rewrote some sections of the doc
- Cleaned up other sections of the doc for specificity/new language and writing guidelines
- Fixed outdated/incorrect doc links

[Project Task Link](https://www.odoo.com/web#id=3878373&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @odoo/inventory-doc-review, this PR is ready for review when one of you has a chance. Thanks!

**Comment by brse-odoo** [comment]
> Hi @odoo/inventory-doc-review, this PR is ready for review when one of you has a chance. Thanks!

Hi @odoo/inventory-doc-review, this PR is ready for review, thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:21]: Nitpick, but this doc only has one H2 (======)
image

For users to tell at a glance what info is in this doc, what do you think about removing the "One, two, and three-step receipts and deliveries" section title, and changing the H3 ones below (One-step flow, Two-step flow, etc) into H2?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:93]: After reading through this doc, I noticed that it doesn't contain specific information, and is more of a comparison doc between the different receipt and delivery types. 

If it's not too much trouble, what do you think of moving the contents of this doc into `content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations.rst`, the parent folder of this doc, like your reservation methods doc! Let me know what you think! :D

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:44]: the acronyms of these removal strategies have yet to be defined on this doc o; I'd spell these out first, in this line (:

**Review by Felicious — APPROVED** [review]
Good job with the rewrite of this doc, @brse-odoo! I had some very minor suggestions for you the consider :smile:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:21]: Good idea! Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:44]: Nice catch. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:93]: I think that's a good idea - I think I'm going to leave this PR as an [IMP] for content-specific updates, though, and once it's merged I'll push up a new restructure to tackle that in a separate PR. Thanks for the suggestion!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:58]:
```suggestion
  by different carriers, or moved to shipping docks before being shipped.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:59]:
```suggestion
- Minimum requirement to use lot numbers, or serial numbers, to track products with a :abbr:`FIFO
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:40]: could be nice to use the ref shorthand for each of these removal strategies like you did below

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:64]:
```suggestion
- Recommended for larger warehouses with high stock levels, or when stocking large items (e.g.
```

**Review by ksc-odoo — APPROVED** [review]
Hey @brse-odoo -- just finished my Final Review. Looks great! Just a small handful of minor suggestions that require your attention. approving now though. so, once those necessary modifications are made -- feel free to tag this for Tech Review

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:8]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:10]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:22]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:27]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:37]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:39]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:43]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:58]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:59]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:64]: Not implemented - comma is required after e.g. when there are multiple items separated by commas. Resolving comment

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/shipments_deliveries.rst:40]: I like to write out the abbreviations upon first mention in the doc, so readers know what they mean, and then do the abbr tag from there on. But, if I get the same feedback again, I will implement! Resolving comment

**Comment by brse-odoo** [comment]
Hi @samueljlieber, this PR is ready for your final lookover, thank you!

**Review by samueljlieber — APPROVED** [review]
Looks good to me @brse-odoo! Nice job 👍
...
@robodoo r+

**Pull request closed without merging**
